src/day8/re_frame/trace.cljs

```diff
@@ -176,10 +178,12 @@
 
 (defn render-trace-panel []
   (let [filter-input     (r/atom "")
-        filter-items     (r/atom [])
+        filter-items     ((fnil #(r/atom %) [])
```

Comment:
This might be a bit hard to read.

```clj
(-> (localstorage/get! "filter-items")
     (or [])
     (r/atom))
```

is clearer, or better still, add a `not-found` arity to `get!` and provide `[]` there. Perhaps even better still is to just handle the nil value for filter-items (if it doesn't cause too many problems).